plugins/frontend/backstage-plugin-wiz/src/components/EntitySeverityChart/SeverityChart.tsx

```diff
@@ -26,13 +26,26 @@ import {
 } from '@backstage/core-components';
 import { BarChart } from '.';
 import { WIZ_PROJECT_ANNOTATION } from '../constants';
+import { useStyles } from '../../style';
+import { Typography } from '@material-ui/core';
 
 export const SeverityChart = () => {
   const api = useApi(wizApiRef);
+  const classes = useStyles();
   const { entity } = useEntity();
   const wizAnnotation =
     entity?.metadata.annotations?.[WIZ_PROJECT_ANNOTATION] ?? '';
 
+  const WizIcon = () => {
+    return (
+      <img
+        src={require('../../../docs/wiz-logo.png')}
```

Comment:
This retrieves the content from outside the `src` folder which isn't something we want to do.